### PR TITLE
Updated Session Numbers/IDs for People Scrapers for 2023 Sessions

### PR DESCRIPTION
**Pull request by bfossen-ce**
The following Jurisdictions needed Session numbers/ids updates for the People Scrapers

AR
GA
IL
MA
ME
MO
MT
SD
WY

### CODE REVIEW

**Review by elseagle — APPROVED** [review]

File: scrapers_next/ma/people.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 
 def list_url():
-    general_court_num = "192"
+    general_court_num = "193"
```
Comment (chrisyamas) [inline scrapers_next/ma/people.py:6]: Would also be good to change it in the example_source for LegDetail on line 13.

(Side note: I tried to run it with the updated value, but I'm getting an SSLCertVerificationError. Is it working for you locally?)

**Review by chrisyamas — CHANGES_REQUESTED** [review]

File: scrapers_next/ma/people.py (hunk 1 above)
Comment (bfossen-ce) [reply scrapers_next/ma/people.py:6]: Yeah I am getting a cert error for both the 192 and 193 sessions. I think MA made some changes. There will be another jira ticket to get this issue sorted out. The URL is the correct one we need though.

File: scrapers_next/ga/people.py [hunk 2]
```diff
@@ -62,7 +62,7 @@ class DirectoryListing(JsonListPage):
     chamber_names = {1: "house", 2: "senate"}
     party_ids = {0: "Democratic", 1: "Republican"}
     source = URL(
-        "https://www.legis.ga.gov/api/members/list/1029",
+        "https://www.legis.ga.gov/api/members/list/1031",
```
[the remark this answers is not in the document]
Comment (bfossen-ce) [reply scrapers_next/ga/people.py:65]: Made the change to the comment.

File: scrapers_next/ar/people.py [hunk 3]
```diff
@@ -92,7 +92,7 @@ def process_page(self):
 
 
 class LegList(HtmlListPage):
-    source = "https://www.arkleg.state.ar.us/Legislators/List?sort=Type&by=desc&ddBienniumSession=2021%2F2021S1#SearchResults"
+    source = "https://www.arkleg.state.ar.us/Legislators/List?sort=Type&by=desc&ddBienniumSession=2023%2F2023S1#SearchResults"
```
[the remark this answers is not in the document]
Comment (bfossen-ce) [reply scrapers_next/ar/people.py]: Thanks, good catch on this one, made the update

File: scrapers_next/ar/people.py [hunk 4]
```diff
@@ -11,7 +11,7 @@ class PartialMember:
 
 
 class LegDetail(HtmlPage):
-    example_source = "https://www.arkleg.state.ar.us/Legislators/Detail?member=B.+Ballinger&ddBienniumSession=2021%2F2021S1"
+    example_source = "https://www.arkleg.state.ar.us/Legislators/Detail?member=B.+Ballinger&ddBienniumSession=2023%2F2023S1"
```
[the remark this answers is not in the document]
Comment (bfossen-ce) [reply scrapers_next/ar/people.py]: Thanks, made this change

**Review by NewAgeAirbender — APPROVED** [review]

**Review by chrisyamas — APPROVED** [review]

**Pull request merged**